MODULE.bazel.lock

```diff
@@ -1,3 +1,17 @@
+# Copyright 2024 The StableHLO Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This is a generated file.

Reply:
I see. I will remove this change. The downstream integration tool should skip any copyright check on files of this ext.